discovery-provider/src/api/v1/tracks.py

```diff
@@ -181,9 +181,7 @@ def get(self):
         if slug and handle:
             routes_parsed.append({"handle": handle, "slug": slug})
 
-        tracks = get_tracks(
-            {"routes": routes_parsed, "with_users": True, "filter_deleted": True}
-        )
+        tracks = get_tracks({"routes": routes_parsed, "with_users": True})
```

Comment:
let's triple check the places we call this API, but this change seems right

Reply:
- RemixSettingsModal: https://github.com/AudiusProject/audius-client/blob/aa685d2174dd5d9457dfb79ac18a5553ba118d5d/src/containers/remix-settings-modal/store/sagas.ts#L40
  - Probably should filter this, don't think we want users remixing deleted tracks?
- RemixesPage: https://github.com/AudiusProject/audius-client/blob/f168424acf90273f59b0e8e51237c82dd8dac057/src/containers/remixes-page/store/sagas.ts#L17
- TrackPage: https://github.com/AudiusProject/audius-client/blob/f168424acf90273f59b0e8e51237c82dd8dac057/src/containers/track-page/store/sagas.js#L114

Reply:
@raymondjacobson ping :)

Reply:
#1 - we definitely want to make sure we filter this client side in future -- but not a huge blocker
#2 - this will allow us to load the `remixes` of a deleted track. i think that's fine
#3 - intention of this PR

lgtm